package.json

```diff
@@ -4,6 +4,7 @@
   "main": "dist/webexWidgets.cjs.js",
   "module": "dist/webexWidgets.esm.js",
   "scripts": {
+    "start": "NODE_ENV=development nodemon src/index.js --exec babel-node",
```

Comment:
I don't think this is how we should do our start script. I would leave this out for a different PR once we agree on the repo structure

Reply:
that was to check if babel works. yes we dont need it.

Reply:
Let's remove it then 👍

Reply:
yes, I'm working on it right now.